runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs

```diff
@@ -290,3 +281,85 @@ pub fn sum_n() {
         value_return(data.len() as u64, data.as_ptr() as u64);
     }
 }
+
+#[no_mangle]
+pub fn insert_strings() {
+    unsafe {
+        input(0);
+        if register_len(0) != 2 * size_of::<u64>() as u64 {
+            panic()
+        }
+        let data = [0u8; 2 * size_of::<u64>()];
+        read_register(0, data.as_ptr() as u64);
+
+        let mut from = [0u8; size_of::<u64>()];
+        let mut to = [0u8; size_of::<u64>()];
+        from.copy_from_slice(&data[..size_of::<u64>()]);
+        to.copy_from_slice(&data[size_of::<u64>()..]);
+        let from = u64::from_le_bytes(from);
+        let to = u64::from_le_bytes(to);
+        let s = vec![b'a'; to as usize];
+        for i in from..to {
+            let mut key = s[(to - i) as usize..].to_vec();
```

Comment:
the line also can be replaced with just creating a new vec of repeated characters instead